File: Servidor/AtHomeTFGCarlosHernandez/src/main/java/org/tfg/athometfgcarloshernandez/spring/model/request/CrearEventoRequestDTO.java
package org.tfg.athometfgcarloshernandez.spring.model.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.tfg.athometfgcarloshernandez.spring.model.EventoDTO;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class CrearEventoRequestDTO {
    private int idCasa;
    private EventoDTO eventoCasa;
    private String fecha;
}

File: Cliente/AtHomeTFGAndroidCarlosHernandez/app/src/main/java/com/example/athometfgandroidcarloshernandez/data/model/request/CrearEventoRequestDTO.kt
package com.example.athometfgandroidcarloshernandez.data.model.request

import com.example.athometfgandroidcarloshernandez.ui.framework.screens.calendario.CalendarioContract

data class CrearEventoRequestDTO (
    var idCasa: String,
    var eventoCasa: CalendarioContract.EventoCasa,
    var fecha:String
)